agents-api/run_dev.py

```
@@ -3,62 +3,77 @@
+import os
 import subprocess
 import sys
-import os
+
 
 def run_command(command, description):
     """Run a command and print description"""
     print(f"🔄 {description}...")
     try:
-        result = subprocess.run(command, shell=True, check=True, capture_output=True, text=True)
+        result = subprocess.run(
```

Ruff (0.13.3)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)